src/platforms/flutter/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -82,3 +80,80 @@ SentryNavigatorObserver(
   setRouteNameAsTransaction: true
 )
 ```
+
+### App Start Instrumentation
+
+<Note>
+
+This feature is available on Sentry Flutter SDK `6.4.0-beta.4` and above. Features available in Beta are still in-progress and may have bugs. We recognize the irony.
+
+</Note>
+
+The App Start Instrumentation provides insight into how long your application takes to launch. It tracks the length of time from the **earliest native process initialization** until the very first [PostFrameCallback](https://api.flutter.dev/flutter/scheduler/SchedulerBinding/addPostFrameCallback.html) is triggered.
+
+The SDK differentiates between a cold and a warm start, but doesn't track hot starts/resumes.
+
+To start getting the Mobile Vitals data, you have to set up the [routing instrumentation](/platforms/flutter/performance/instrumentation/automatic-instrumentation/#routing-instumentation).
```

Comment:
```suggestion
```

@denrase this is not the case for App start anymore, right, since we use Event Processors.